packages/gatsby-cli/src/init-starter.js

```diff
@@ -8,11 +9,14 @@ const report = require(`./reporter`)
 const url = require(`url`)
 const existsSync = require(`fs-exists-cached`).sync
 const { trackCli, trackError } = require(`gatsby-telemetry`)
+const prompts = require(`prompts`)
 const spawn = (cmd: string, options: any) => {
   const [file, ...args] = cmd.split(/\s+/)
   return execa(file, args, { stdio: `inherit`, ...options })
 }
 
+const conf = new Configstore(`gatsby`, {}, { globalConfigPath: true })
```

Comment:
@jamo Can you check if this affects our conditional in telemetry?

Reply:
It's a bit tricky to verify, but based on one two tests and one simulation co-existence of multiple configstores is fine. And as long as we don't poke the `telemetry` namespace we should be good.

Reply:
How could I be sure to not make a regression on this ?

Reply:
Reading the configstore core and based on manual testing I'm fairly confident that we can safely have multiple configstore instances w/o any issues or regressions